source/_docs/scripts/service-calls.markdown

```diff
@@ -176,6 +176,87 @@ data:
 
 {% endraw %}
 
+### Use response data in templates: convert YAML to JSON
+
+The response data is returned in Yaml, and can not be used as is in the template editor.
```

Comment:
```suggestion
The response data is returned in YAML and can not be used as is in the template editor.
```

I'm not sure why this matters?

Reply:
I added that because in the response after the new service was implemented, users did not understand what happened, or how they could use that response.
understanding the type of data is the beginning of the way to the answer, so seems important